ui/src/components/common/AppItem.jsx

```diff
@@ -20,14 +20,29 @@ class AppItem extends PureComponent {
         }}
       />
     );
-    return (
+    const ftm_message = (
```

Comment:
I know it’s slightly confusing, but we use camelCase in JS land ;)

```suggestion
    const ftmMessage = (
```